File: app/src/main/java/com/plbear/iweight/model/settings/SettingsActivity.java
package com.plbear.iweight.model.settings;

import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import com.plbear.iweight.R;
import com.plbear.iweight.base.BaseActivity;

import butterknife.BindView;

/**
 * Created by yanyongjun on 2018/6/11.
 */

public class SettingsActivity extends BaseActivity {
    @BindView(R.id.lab_title)
    TextView lab_title;

    @BindView(R.id.btn_details_select_all)
    Button btn_details_select_all;

    @Override
    public int getLayout() {
        return R.layout.activity_settings;
    }

    @Override
    public void afterLayout() {
        init();
    }

    public void init(){
        lab_title.setText(R.string.settings);
        btn_details_select_all.setVisibility(View.GONE);
    }



    public static String PREFERENCE_KEY_SET_TARGET_WEIGHT = "set_target_weight";
    public static String PREFERENCE_KEY_ONLY_ONCE_EVERYDAY = "only_once_everyday";
    public static String PREFERENCE_KEY_EXPORT_IMPORT = "export_import_switch";
    public static String PREFERENCE_KEY_UNIT = "value_unit";
}

File: app/src/main/java/com/plbear/iweight/model/settings/SettingsActivity.kt
package com.plbear.iweight.model.settings

import android.view.View

import com.plbear.iweight.R
import com.plbear.iweight.base.BaseActivity
import kotlinx.android.synthetic.main.include_title.*

/**
 * Created by HuHu on 2017/normal_3/13.
 */

class SettingsActivity : BaseActivity() {

    override fun getLayout(): Int {
        return R.layout.activity_settings
    }

    override fun afterLayout() {
        init()
    }

    fun onClick_back(v:View){
        finish()
    }

    private fun init() {
        lab_title.setText(R.string.settings)
    }

    companion object {
        val PREFERENCE_KEY_SET_TARGET_WEIGHT = "set_target_weight"
        val PREFERENCE_KEY_ONLY_ONCE_EVERYDAY = "only_once_everyday"
        val PREFERENCE_KEY_EXPORT_IMPORT = "export_import_switch"
        public val PREFERENCE_KEY_UNIT = "value_unit"

        private val TAG = "SettingsActivity"
    }
}
